.github/workflows/build.yml

```
@@ -45,7 +45,7 @@ jobs:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
           tags: |
             type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', github.event.repository.default_branch) }}
-            type=raw,value={{date 'YYYYMMDD-HHmmss'}}-{{sha}}
+            type=raw,value={{date 'YYYYMMDDTHHmmssZ'}}-{{sha}}
```

ISO 8601 date strings have hyphens between the date parts and colons between the time parts.

Given those two separators are already used, I suggest an underscore to separate the sha.

```suggestion
            type=raw,value={{date 'YYYY-MM-DDTHH:mm:ssZ'}}_{{sha}}
```